web-ts/src/hocs/withPlatform.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
```

Comment:
No, because it is no component, it is a function which returns a componen. Should be refactored to a hook and be put into `src/hooks` in the future though